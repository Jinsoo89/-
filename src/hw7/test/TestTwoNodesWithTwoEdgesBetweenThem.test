# Test FindMinCostPath with two nodes and two edges between them (One way)

# create graph with two nodes, and two edges (One way)
CreateGraph players
AddNode players C.Ronaldo
AddNode players W.Rooney
AddEdge players C.Ronaldo W.Rooney 0.1
AddEdge players C.Ronaldo W.Rooney 1.0

# find path between two nodes
FindPath players C.Ronaldo W.Rooney