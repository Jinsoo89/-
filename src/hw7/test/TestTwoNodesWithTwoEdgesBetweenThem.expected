# Test FindMinCostPath with two nodes and two edges between them (One way)

# create graph with two nodes, and two edges (One way)
created graph players
added node C.Ronaldo to players
added node W.Rooney to players
added edge 0.100 from C.Ronaldo to W.Rooney in players
added edge 1.000 from C.Ronaldo to W.Rooney in players

# find path between two nodes
path from C.Ronaldo to W.Rooney:
C.Ronaldo to W.Rooney with weight 0.100
total cost: 0.100